crates/ruff/src/rules/flake8_pyi/rules/simple_defaults.rs

```diff
@@ -188,10 +193,18 @@ pub fn typed_argument_simple_defaults(checker: &mut Checker, args: &Arguments) {
             {
                 if arg.node.annotation.is_some() {
                     if !is_valid_default_value_with_annotation(default, checker) {
-                        checker.diagnostics.push(Diagnostic::new(
-                            TypedArgumentSimpleDefaults,
-                            Range::from(default),
-                        ));
+                        let mut diagnostic =
+                            Diagnostic::new(TypedArgumentSimpleDefaults, Range::from(default));
+
+                        if checker.patch(diagnostic.kind.rule()) {
+                            diagnostic.amend(Fix::replacement(
+                                "...".to_string(),
+                                default.location,
+                                default.end_location.unwrap(),
```

Comment:
Can we add a `SAFETY: <reason>` comment here and on line 234 explaining why it's safe to call `unwrap` here?

Reply:
This is done in a lot of places in the code.
I do not think it is needed.
`node.end_location` is always the `Some` variant (c.f. RustPython/RustPython#4192)